tests/PanelTest.php

```diff
@@ -77,6 +77,18 @@ public function testGetTraceLine_CustomLinkByCallback_CustomText()
         };
         $this->assertEquals('<a href="ide://open?url=file.php&line=10">custom text</a>', $panel->getTraceLine($traceConfig));
     }
+    
+    public function testGetTraceLine_traceBasePath()
+    {
+        $filePath = \Yii::$app->basePath . '/file.php';
```

Comment:
I'm not sure if this is the right approach to this test. Wouldn't mind help if there's a better way.

Reply:
Please use camelCase notation. Smth like testGetTraceLineWithTraceBasePath

Reply:
I'm matching the other test cases function names.